docs/proposals/PROPOSAL_C.md

```diff
@@ -0,0 +1,295 @@
+# Proposal C
+
+Create a Node manifest.
+
+## Description
+
+The proposal introduces a "Node" manifest used to organize container images with related artifacts or to group container images together into a higher order artifact. It is based on the typical "Node" data structure used for creating linked lists.
+
+A detailed description of this proposal can be found [here](https://docs.google.com/document/d/1KYJaCm5Z-BJ7hVOI_lDh4vEvrmyEAMwUp2nJBELstGw/edit).
+
+## Links
+
+| Description                                 | Link                        |
+| ------------------------------------------- | --------------------------- |
+| A talk that introduces the idea   	      | [View](https://youtu.be/U_Q5bG-G_Pw) |
+| A Detailed description with pictures        | [View](https://docs.google.com/document/d/1KYJaCm5Z-BJ7hVOI_lDh4vEvrmyEAMwUp2nJBELstGw/edit) |
+
+## Modifications
+
+No modifications to existing JSON schemas or APIs are required. A new object is introduced instead.
```

Comment:
Perhaps just drop this line.. There are indeed new endpoints introduced in the registry section, but reader can go there to learn more

Reply:
or rather, place it under "JSON Schema"